src/Entity/Organization.php

```
@@ -0,0 +1,268 @@
+
+        return $this;
+    }
+
+    public function isPersonal(): bool
+    {
+        return $this->isPersonal;
+    }
+
+    public function getSettings(): OrganizationSettings
+    {
+        return OrganizationSettings::fromArray($this->settings);
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Entity\OrganizationSettings' in method 'getSettings'. (undefined)

(StaticAccess)